manifests/opendatahub/overlays/visualization-server/kustomization.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  # Build Configs
+  - ./buildconfigs/visualization-server.yaml
+
+  # CustomResourceDefinitions
+  - ./customresourcedefinitions/viewers.yaml
+
+  # Deployments
+  - ./deployments/ds-pipeline-viewer-crd.yaml
```

Comment:
if we are to move the viewer crd into overlays, I feel like it should be it's own separate overlay as it is a decoupled service with its own deployments/services/etc.

Reply:
See my previous comment. Viz Server + Viewer CRD composes a separate component. One cannot work without the other, which makes sense to me keeping them in the same overlay.